### PR TITLE
Upload autoupg.xml in reboot_and_wait_up_upgrade module

### PR DESCRIPTION
**Pull request by waynechen55**
* **It** seems that reboot_and_wait_up_upgrade has different base module to date back if failure happens. So upload autoupg.xml file here explicitly.
* **Related ticket:** n/a
* **Needles:** n/a
* **Verification run:**
  No need actually, but verified by using perl online compiler

### CODE REVIEW

**Comment by waynechen55** [comment]
You can have a look @alice-suse @guoxuguang @Julie-CAO

File: tests/virt_autotest/reboot_and_wait_up_upgrade.pm [hunk 1]
```diff
@@ -53,7 +53,10 @@ sub post_fail_hook {
     my ($self) = shift;
     reset_consoles;
     select_console('root-console');
-    $self->upload_y2logs if (check_var('offline_upgrade', 'yes'));
+    if (check_var('offline_upgrade', 'yes')) {
+        upload_logs("/root/autoupg.xml", failok => 1);
```
Comment (Julie-CAO) [inline tests/virt_autotest/reboot_and_wait_up_upgrade.pm:57]: I heard from Leon that /root/autoupg.xml does not exist during upgrade. I might not know the detail, @guoxuguang  may give some input here.

Comment (alice-suse) [reply tests/virt_autotest/reboot_and_wait_up_upgrade.pm:57]: It is already uploaded in the earlier step, after the /root/autoupg.xml is generated.

Comment (waynechen55) [reply tests/virt_autotest/reboot_and_wait_up_upgrade.pm:57]: I am fine if this has been uploaded.

**Comment by waynechen55** [comment]
Double confirmed this file can not be accessed during eboot_and_wait_up_upgrade. Closing.

**Pull request closed without merging**